backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/CassandraTableManager.java

```diff
@@ -45,17 +45,16 @@ public CassandraTableManager(CassandraModule module, Session session) {
         this.module = module;
     }
 
-    public CassandraTableManager ensureAllTables() {
+    public boolean ensureAllTables() {
```

Comment:
Looking at that method it is not clear what information this boolean carry.

Is it `AtLeastOneTableCreated` ?